lib/kitchen/driver/ec2.rb

```diff
@@ -341,6 +342,15 @@ def block_device_mappings
 
         bdms
       end
+
+      # TODO Update this to use the new Transport logic when that is ready
+      def create_ec2_json(state)
+        Kitchen::SSH.new(*build_ssh_args(state)) do |conn|
+          run_remote("sudo mkdir -p /etc/chef/ohai/hints;sudo touch /etc/chef/ohai/hints/ec2.json",
```

Comment:
Will sudo always be available? I think we use it elsewhere but I may misremember.